tests/DefaultsSchemaValidatorTest.php

```diff
@@ -70,22 +70,23 @@ public function testBooleanCoerceFalse(): void {
   }
 
   public function testNestedCoerceFalse(): void {
+    // @TODO This testcase does nothing. $cases is an unused variable.
```

Comment:
Hahaha, HHAST can detect unused locals. HHAST was right here. $cases is not being tested.